### ISSUE TITLE
support for <webview>

### ISSUE THREAD

**Issue by miaulightouch** [GoogleChromeLabs/carlo#139]
either carlo nor puppeteer didn't support <webview> tag, is any method to get it work?

**Comment by LongTengDao** [comment]
Web Components?

**Comment by miaulightouch** [comment]
https://developer.chrome.com/apps/tags/webview <webview> tag from chrome apps

**Comment by tracker1** [comment]
Would IFRAME work for you?

**Comment by miaulightouch** [comment]
nope, I have to interact directly with web page for more control.
